src/components/SecurityPage.jsx

```diff
@@ -64,28 +65,28 @@ const DumbSecurityPage = props => {
       <NarrowContent>
         <Stack>
           <img src={strongPasswordIcon} alt="" width="204" />
-          <MainTitle>Plus qu&apos;un seul mot de passe à retenir</MainTitle>
+          <MainTitle>{t('SecurityPage.title')}</MainTitle>
           <Stack spacing="xxl">
-            <Text>
-              Le mot de passe de votre Cozy devient l&apos;unique clé pour
-              décoder vos mots de passe sauvegardés de manières sécurisée
-            </Text>
+            <Text>{t('SecurityPage.description')}</Text>
```

Comment:
Do not forget to fix "manières" in the French translation (I guess it's in transifex ?)

Reply:
Translations are plain json files for now. « Manières » is already fixed in the french one ;) Thanks